docs/admission-control-secure.md

```diff
@@ -109,39 +109,39 @@ spec:
       args:
         - "--replicate-cluster=v1/namespaces"
         - "--replicate=extensions/v1beta1/ingresses"
-        - "--opa-auth-token=kube-mgmt"
+        - "--opa-auth-token-file=/policy/token"
```

Comment:
I think this should be `/policies/token` to match the volumeMount below.

Reply:
ah. good catch. thanks.